src/extensionFunctions.ts

```diff
@@ -898,3 +903,54 @@ export async function copyFileTextToClipboard(uri: vscode.Uri) : Promise<void> {
 		}
 	}
 } 
+
+export async function pasteClipboardToActiveEditor() : Promise<void> {
+	let currentEditor : vscode.TextEditor | undefined = vscode.window.activeTextEditor;
+	if (!currentEditor) {
+		await vscode.commands.executeCommand('workbench.action.openPreviousRecentlyUsedEditor');
```

Comment:
sounds like it is not only the Active text Editor but also the previously used editor

Reply:
I don't see that as a bad thing, but definitely something to hit in the docs

Reply:
The name of the function and what is effectively achieved by this function are not matching

Reply:
Github did not update the PR flow for some reason. But if you look at commit https://github.com/redhat-developer/vscode-didact/pull/267/commits/fc395eec4b86d42554a549d9d783ba30973b3b83, you will see that the function is now named "pasteClipboardToActiveEditorOrPreviouslyUsedOne" which accurately matches what's going on in the function